packages/web-components/src/menu/menu.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorTransparentStroke,
+  shadow16,
+} from '../theme/design-tokens.js';
+
+/** Menu styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    display: flex;
```

Comment:
Isn't this covered by the `display()` utility?
```suggestion
```

Reply:
Yes, it is :)